query/common/src/main/java/org/codice/ddf/admin/common/fields/common/AddressField.java

```diff
@@ -36,14 +36,37 @@ public AddressField() {
         super(DEFAULT_FIELD_NAME, FIELD_TYPE_NAME, DESCRIPTION);
     }
 
+    public AddressField hostname(String hostname) {
```

Comment:
Explore having only AddressField and not UrlField to normalize the requests coming in.